server/constants/user-roles.ts

```diff
@@ -0,0 +1,12 @@
+export enum Role {
```

Comment:
I wanted to update all string roles with this enum, but I am going to do that in another ticket cause it'll cause too many files changes